app/reducers/storage/index.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {combineReducers} from 'redux';
+import {StorageTypes} from 'app/constants';
+
+function info(state = {}, action) {
```

Comment:
I don't think we want another top level store for this. I think we should just be loading and setting the state based off the contents of storage, not providing a separate store that maps to local storage. Eventually we'll start serializing more than just the token to the local storage and then doing it this way won't make sense